packages/expo-image/ios/EXImageView.m

```diff
@@ -28,16 +36,27 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
     _bridge = bridge;
     _needsReload = NO;
     _resizeMode = RCTResizeModeCover;
-    self.contentMode = [EXImageTypes resizeModeToContentMode:_resizeMode];
     _intrinsicContentSize = CGSizeZero;
+    _cornerRadii = EXImageCornerRadiiInit();
+    _borders = EXImageBordersInit();
+    _borderLayers = @{};
+
+    _imageView = [SDAnimatedImageView new];
```

Comment:
We could use `alloc] initWithFrame:self.bounds]`

Reply:
Good suggestion, will do 👍